pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md

```diff
@@ -79,4 +86,4 @@ The brand's Authorization Service should have to register configuration of LiveP
 
 *	LP Popup redirect URI (when external web window is used)
```

Comment:
Nit, but please use only a space between asterisk and text (i.e., “* Text”).

Reply:
@j9t - I don't understand

Reply:
For some reason these lists use a tab between asterisk and text of each list item entry—note, for example, the extraordinarily large space between “*” and “LP Popup”.

<img width="124" alt="Screen Shot 2022-11-21 at 10 24 04" src="https://user-images.githubusercontent.com/1700772/203013827-aa16a486-4a64-4f88-9b71-ba5a8a34b71c.png">

Please use a single space instead.